lib/CMakeLists.txt

```diff
@@ -54,6 +54,12 @@ set (LIBS
     ${LEATHERMAN_LIBRARIES}
 )
 
+if (WIN32)
+    # Necessary when statically linking cpp-pcp-client on Windows.
+    # Shouldn't hurt when cpp-pcp-client is a DLL.
```

Comment:
I'm not sure there's a way to tell from here. /cc @branan 

I could add a CMake config file to the cpp-pcp-client project that gets installed, and would replace the Findcpp-pcp-client script in this project. That would include link dependencies and whether it's static or dynamic. However there's pretty much no downside with linking an extra system library.